gtdynamics/universal_robot/sdf.cpp

```diff
@@ -203,15 +224,17 @@ static LinkJointPair ExtractRobotFromSdf(const sdf::Model &sdf) {
     if (parent_link_name == "world") {
       // This joint fixes the child link in the world frame.
       LinkSharedPtr child_link = name_to_link[child_link_name];
-      Pose3 fixed_pose = child_link->wTcom();
+      Pose3 fixed_pose = child_link->bTcom();
       child_link->fix(fixed_pose);
       continue;
     }
     LinkSharedPtr parent_link = name_to_link[parent_link_name];
     LinkSharedPtr child_link = name_to_link[child_link_name];
+    sdf::Link parent_sdf_link = *sdf.LinkByName(parent_link_name);
```

Comment:
Hmmm. Why are we de-referencing the pointers here? Why not simply pass the pointers to the function?

Reply:
fixed, done